components/calendar/test/calendar.test.js

```diff
@@ -92,6 +93,27 @@ describe('d2l-calendar', () => {
 			const expectedFocusDate = new Date(2015, 8, 2);
 			expect(calendar._focusDate).to.deep.equal(expectedFocusDate);
 		});
+
+		it('dispatches d2l-calendar-view-change event when user changes to previous month', async() => {
+			const calendar = await fixture(normalFixture);
+			const el = calendar.shadowRoot.querySelectorAll('d2l-button-icon')[0];
+			setTimeout(() => el.click());
+			const { detail } = await oneEvent(calendar, 'd2l-calendar-view-change');
+			await aTimeout(1);
```

Comment:
Curious why this was needed since I'd expect the detail to be there immediately?

Reply:
Good question. I copied that from another test and forgot to ditch it. Removing... Thanks!